### DISCUSSION TITLE
Support request:  Not working when opening project from `zed .` command !

### DISCUSSION THREAD

**Discussion by rojanmagar2001** [aws/codewhisperer-command-line-discussions#81] [Support ticket] [answered]
### Checks

- [X] I ran `cw doctor` in the affected command line session and it didn't resolve my issue
- [X] I ran `cw restart` and it didn't resolve my issue
- [X] I searched [the discussions in this repo](https://github.com/aws/codewhisperer-command-line-discussions/discussions?q=) and there are no duplicates of my issue

### Expected behaviour

Should work when opening with zed . also

### Actual behaviour

cwterm not enabled by default when opening with terminal.

**Comment by brendanfalk** [comment]
Hey! Could you please clarify the issue a little bit more? Is the issue that it usually works in the zed terminal, but it's not working when you run `zed .`?

**Reply by rojanmagar2001** [reply, marked as answer]
when opening project from the terminal with `zed .` command. The cwterm was not working but it's working now. Thanks.
